dashboard/app/models/ability.rb

```diff
@@ -318,12 +318,20 @@ def initialize(user)
       end
     end
 
+    # Checks if user is directly enrolled in pilot
     if user.persisted?
       if Experiment.enabled?(user: user, experiment_name: 'csa-pilot')
         can :get_access_token, :javabuilder_session
       end
     end
 
+    # Checks if user has a teacher enrolled in pilot
+    if user.persisted?
```

Comment:
can you add a test to `javabuilder_sessions_controller_test` that verifies this permission works? (we already have a test that a random student cannot access a token)

Reply:
Good call- done, thanks!